src/game_state/player.rs

```diff
@@ -138,10 +145,45 @@ fn handle_message(
                 .unwrap();
             }
         }),
+        PlayerStateOperations::CrossBorder(msg) => {
+            let player = players
+                .get(&msg.local_conn_id)
+                .expect("Could not cross border: player not found");
+            broadcast_packet!(
+                messenger,
+                Packet::DestroyEntities(player.destroy_entities()),
+                Some(player.conn_id),
+                true
+            )
+            .unwrap();
+            send_packet!(
+                messenger,
+                msg.remote_conn_id,
+                Packet::PlayerPositionAndLook(player.player_position_and_look())
```

Comment:
For the border crossing login, we just send the player position and look packet. Eventually we will need our own packet that sends other info like username, etc